charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/elasticsearch-logging-service-account.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: elasticsearch-logging
+  namespace: {{ .Release.Namespace }}
+  labels:
+    gardener.cloud/role: logging
```

Comment:
Please, add the old label key `garden.sapcloud.io`, too.
It is needed for the deletion of the logging resources.